lib/storage-client.js

```diff
@@ -246,6 +246,8 @@ class ChromedriverStorageClient {
     } finally {
       await fs.rimraf(tmpRoot);
     }
+    await fs.chmod(dst, 0o755);
```

Comment:
It makes sense to move permissions change out of this method, since it has nothing to do with unzipping and put it into retrieveDriver method, right next to `await this.unzipDriver(archivePath, targetPath)` call

Reply:
hmm, ok 👍